### DISCUSSION TITLE
Avalon: Where is the severity fatal (FacesMessage.SEVERITY_FATAL)?!

### DISCUSSION THREAD

**Discussion by marcelocaser** [primefaces/community#3072] [PrimeFaces Templates] [answered]
Hi, 

I was using the PF 8 version with the Avalon 2.1.2 theme and in this version there is the 'Severity Fatal' option, as shown in the image below:

![image](https://github.com/user-attachments/assets/80963515-406a-4483-9382-0f60955b72e9)

I noticed that in the latest versions of PF 11 or higher, and in the 'Avalon 3.x' or higher theme and even in the 'PF 14 showcase', there is no longer the 'Severity Fatal' option. See image below:

![image](https://github.com/user-attachments/assets/a626b279-b198-4c09-8966-eb568212e2f2)

The funny thing is that in the documentation for all PF 11 or higher versions, it mentions the option.

![image](https://github.com/user-attachments/assets/b356ba0d-9f61-45e8-a03b-3747df0b86f4)

**Comment by melloware** [comment]
Well FATAL still exists are you just saying it was removed from the Avalon template?   because you can use FATAL just fine in JSF.

**Comment by marcelocaser** [comment]
I think I expressed myself badly. Yes, it was removed from the ‘Avalon’ theme and I checked in other themes like: ‘Freya’, ‘Ultima’… and there is no FATAL.

There is also no FATAL option in the showcase.

**Reply by melloware** [reply, marked as answer]
I have added it back to the showcase: https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/message/messages.xhtml

PR for PF showcase will be in 15.0.0

**Reply by marcelocaser** [reply]
Nice, perhaps a customization similar to the PF 8 version would be interesting, such as the color and icon used.

PF 8:
Message
![image](https://github.com/user-attachments/assets/229034be-54f9-4e34-9321-90c0a1a4aa05)

StaticMessage
![image](https://github.com/user-attachments/assets/e90b22a5-50b5-4df3-a0cf-d3ebbe46db52)

**Reply by melloware** [reply]
PrimeFaces 15.0.0

![image](https://github.com/user-attachments/assets/375dd399-d88c-430c-806f-a96c7eea1320)

**Comment by tandraschko** [comment]
it seems it was removed because error/fatal has the same styling in the templates

i fixed now the icon for upcoming versions but it was a design decision propably that error/fatal look the same
